README.md

```diff
@@ -133,18 +133,16 @@ If you want to explicitly invoke a `const` constructor, you can use the followin
 Note the `@` before `const`. This is because `@const` is an annotation, not a keyword.
 The Kotlin compiler does not support keywords in front of expressions at the parser level.
 
-The difference is easy to remember, with any _declaration_ you must use `const`, and with any
+The difference is easy to remember: with any _declaration_ you must use `const`, and with any
 _invocation_ you must use `@const`.
 
-Note that `@const` is not necessary when it's implied by e.g. assigning to a `const val`,
-similar to Dart.
+Note that similar to in Dart, `@const` is not necessary when it's implied, e.g. by assigning to a `const val`.
```

Comment:
```suggestion
Note that similar as in Dart, `@const` is not necessary when it's implied, e.g. by assigning to a `const val`.
```
I think this is more correct?

Reply:
Changed `similar to in` to just `as in`, which I think flows a bit better anyway.